docs/apigateway.md

```diff
@@ -141,14 +147,10 @@ Action: putBooks
   Path: /books
   Verb: put
   URL: https://${APIHOST}:9001/api/21ef035/club/books
-Action: deleteBooks
-  API Name: Book Club
-  Base path: /club
-  Path: /books
-  Verb: delete
-  URL: https://${APIHOST}:9001/api/21ef035/club/books
 ```
 
+*Note:* APIs will be sorted alphabetically by default, first by Base path, then by Path (relative path), then by Verb. If we need to sort alphabetically by Action name, we can add the `-n` or `--sort-action` flags to do so.
```

Comment:
Remove "we" from the second sentence. Ie, "In order to sort alphabetically by Action name, add the `-n` or `--sort-action` flags."